sceptre/plan/plan.py

```diff
@@ -35,6 +35,7 @@ def __init__(self, context):
 
         config_reader = ConfigReader(context)
         all_stacks, command_stacks = config_reader.construct_stacks()
+        self.read = config_reader.read(self.context.command_path)
```

Comment:
Why not add a `read()` method instead that would do this instead? Also, what happens if the `command_path` is a StackGroup? I suspect that will actually fail.

Reply:
What you _could_ do is `read_config()` method on the Actions class that would be like this:

```python
def read(self, config_reader):
    return config_reader.read(self.stack.name)
```

^^ I think that might work. That way, SceptrePlan would ultimately map out to actions and operate on StackGroups or individual Stacks correctly, like it does for everything else. You could pass the ConfigReader as a method parameter to the Actions (since it won't have access to that otherwise).